aospy/utils/times.py

```diff
@@ -472,6 +397,9 @@ def _assert_has_data_for_time(da, start_date, end_date):
     AssertionError
```

Comment:
Above in this docstring: netCDF4.netcdftime -> cftime (maybe grep for the former throughout the package)

Reply:
Good catch.  This turns out to be the only place (and I fixed it).